packages/sdk/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
nit: we should drop this license. There's a license at the base of the repo that is not MIT.